src/Storage/Storage.Management/Blob/GetAzureStorageBlobServiceProperties.cs

```diff
@@ -0,0 +1,109 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.Storage
+{
+    using Microsoft.Azure.Commands.Management.Storage.Models;
+    using Microsoft.Azure.Management.Storage;
+    using Microsoft.Azure.Management.Storage.Models;
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+    /// <summary>
+    /// Modify Azure Storage service properties
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + StorageBlobServiceProperty, DefaultParameterSetName = AccountNameParameterSet), OutputType(typeof(PSBlobServiceProperties))]
```

Comment:
Shouldn't thsi just be 'Get-AzStorageBlobService'?

Reply:
Blob Service support Container, blob, blob service priperties. Add properties might be more specific, and this is also aligned with dataplane cmdlet name: Get-AzureStorageServiceProperties

Reply:
As we discussed ,will still use Get-AzureStorageServiceProperty